src/screens/Product/Product.js

```diff
@@ -1,17 +1,26 @@
 import React, { Component } from 'react';
 import { Text, View, ScrollView } from 'react-native';
+import LottieView from 'lottie-react-native';
 import CustomButton from 'src/components/CustomButton';
 import styles from './styles';
 
 class Product extends Component {
   navigateToList = () => this.props.navigation.navigate('ProductsList');
- render () {
+  render () {
     const { navigation } = this.props;
     const textObj = navigation.getParam('custom_attributes').find(item => item.attribute_code === 'description');
```

Comment:
add default for getParam `navigation.getParam('custom_attributes', [])`. I've got crash while launching ))